azure-mgmt-resource/azure/mgmt/resource/policy/v2017_06_01_preview/operations/policy_definitions_operations.py

```diff
@@ -290,3 +486,76 @@ def internal_paging(next_link=None, raw=False):
             return client_raw_response
 
         return deserialized
+
+    def list_by_management_group(
+            self, management_group_id, filter=None, custom_headers=None, raw=False, **operation_config):
```

Comment:
The filter param should go away with my changes in [this ](https://github.com/Azure/azure-rest-api-specs/pull/1788)PR.